pkg/arvo/app/chat-cli.hoon

```diff
@@ -111,13 +109,15 @@
 ::
 %-  agent:dbug
 %+  verb  |
-^-  agent:gall
+%-  (agent:shoe command)
+^-  (shoe:shoe command)
 =<
   |_  =bowl:gall
   +*  this       .
       talk-core  +>
       tc         ~(. talk-core bowl)
       def        ~(. (default-agent this %|) bowl)
+      des        ~(. (default:shoe this command) bowl)
```

Comment:
Might be reasonable to use `default:shoe` as a total replacement for `default-agent` so you don't have to think about whether an arm comes from `+agent` or `+shoe`.

Reply:
Hmm, I've been going back and forth on whether I like that or not. Leaning towards not liking it.

Having to think about the distinction between the `+agent` and `+shoe` arms doesn't seem like the biggest deal, and I kind of like the emphasis on the "standard" vs "non-standard" separation.

(And yes, technically all of these arms come from `+shoe`, because they have the augmented `card` type... but the `+agent` ones still nest fine.)

Ultimately I don't think this library has any business saying anything about regular gall agent arms (outside of the cases it was designed to handle), even if that's just saying "look at these widely used defaults". The type system (wrt cores) already forces this into knowing more about `+agent` than it wants to, let's not add insult to injury.

This has been entirely too many words, because I don't feel very strongly about this, and it doesn't sound like you do either. Happy to discuss further, or let it be.

Reply:
I don't feel strongly either, let's leave it.